core/vm/offchain.go

```diff
@@ -0,0 +1,9 @@
+package vm
+
+import (
+	"github.com/ethereum/go-ethereum/suave/offchain"
+)
+
+type System interface {
+	Blocks() offchain.Blockstore
```

Comment:
why is this using `Blocks` and `Blockstore`? wouldn't this be kind of a generic store? the name implies it's for blocks 🤔

Reply:
Yeah, it's confusing.  I'll change this to something like `Blob`, which is more accurate.